server/channels/db/migrations/postgres/000120_create_channelbookmarks_table.up.sql

```diff
@@ -0,0 +1,21 @@
+CREATE TABLE IF NOT EXISTS channelbookmarks (
+    id varchar(26) PRIMARY KEY,
+    ownerid varchar(26) NOT NULL,
+    channelid varchar(26) NOT NULL,
+    fileinfoid varchar(26) DEFAULT NULL,
+    createat bigint DEFAULT 0,
+    updateat bigint DEFAULT 0,
+    deleteat bigint DEFAULT 0,
+    displayname text DEFAULT '',
+    sortorder integer DEFAULT 0,
+    linkurl text DEFAULT NULL,
+    imageurl text DEFAULT NULL,
+    emoji varchar(64) DEFAULT NULL,
+    type varchar(26) DEFAULT 'link',
```

Comment:
You can create an enum type as `CREATE TYPE bookmark_type AS ENUM ('link', 'file');` and then use `bookmark_type` as the column type. See `create_enums.up.sql`